pkgs/applications/misc/sampler/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoModule, fetchFromGitHub, alsaLib }:
+
+buildGoModule rec {
+  pname = "sampler";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "sqshq";
+    repo = "sampler";
+    rev = "v${version}";
+    sha256 = "129vifb1y57vyqj9p23gq778jschndh2y2ingwvjz0a6lrm45vpf";
+  };
+
+  modSha256 = "0wgwnn50lrg6ix5ll2jdwi421wgqgsv4y9xd5hfj81kya3dxcbw0";
+
+  subPackages = [ "." ];
+
+  buildInputs = [ alsaLib ];
+
+  meta = with stdenv.lib; {
```

Comment:
`stdenv` isn't used anywhere else so you could just have `lib` as an argument at the top and put `with lib;` here